.circleci/config.yml

```diff
@@ -5,8 +5,8 @@ jobs:
    docker:
      - image: circleci/android:api-26-alpha
    environment:
-     JVM_OPTS: -Xmx3200m
-     CIRCLE_JDK_VERSION: oraclejdk8
+     - JVM_OPTS: -Xmx3200m
```

Comment:
Not sure if this syntax is valid (leading `-`). Any reason to not use what's in analytics-android? https://github.com/segment-integrations/analytics-android-integration-adobe-analytics/pull/65/files. That's the example from the docs as well.

https://circleci.com/docs/2.0/env-vars/

```
   environment:
          POSTGRES_USER: conductor
          POSTGRES_DB: conductor_test
```

Reply:
It looks like at least example includes the leading hyphen: https://www.dropbox.com/s/trcswb4iuxcdz1f/Screenshot%202018-01-23%2013.40.31.png?dl=0

Let me rebuild without to validate whether the issue here was also just a flaky error.